src/Test/L0/Worker/CodeCoverage/CodeCoverageConstants.cs

```diff
@@ -0,0 +1,208 @@
+﻿namespace Microsoft.VisualStudio.Services.Agent.Tests.Worker.CodeCoverage
+{
+    public class CodeCoverageConstants
+    {
+        public static readonly string ValidJacocoXml =
```

Comment:
I didnot find a deployment attribute equivalent in xunit test. Hence made them const.